.mandatory_files

```diff
@@ -24,5 +24,10 @@ tests/fixtures/__init__.py
 Dockerfile
 config_schema.json
 example_config.yaml
+LICENSE
 README.md
 setup.py
+setup.cfg
+requirements.txt
+requirements-dev.txt
+requirements-dev-common.txt
```

Comment:
Don't need add files that are already in the static list.
```suggestion
requirements-dev.txt
```

Reply:
The idea was to make sure they exist, since the static list can be overridden.